SPECS/privacyidea.spec

```diff
@@ -47,6 +47,13 @@ mkdir -p %{_tmp_build_dir}
 git clone --recurse-submodules --branch v%{version} --depth 1 https://github.com/privacyidea/privacyidea.git %{_tmp_build_dir}/privacyidea
 
 %build
+#  npm ci && ng build
+pushd %{_tmp_build_dir}/privacyidea/privacyidea/static_new
+npm ci
+npm run-script ng build
+rm -rf node_modules
```

Comment:
We could use something like:
```
find . -maxdepth 1 ! -name 'dist' ! -name '.' -exec rm -rf {} +
```
to remove all unnecessary files and folders from `static_new`